packages/lib/testing/test-utils.ts

```diff
@@ -1042,4 +1043,9 @@ export const newOcrService = () => {
 	return new OcrService(driver);
 };
 
+export const cleanUpClientDb = async () => {
+	const allResourceIds = await Resource.all({ fields: ['id'] });
+	await Resource.batchDelete(allResourceIds.map(r => r.id));
+};
+
```

Comment:
Calling this, should have the same effect:

await setupDatabaseAndSynchronizer(1);
		await switchClient(1);

Reply:
I removed this since function since this is the way that is done in the rest of the project and the test file now has a beforeEach that resets the database.